trade_file.md

```diff
@@ -132,6 +132,7 @@ This trade represents a trade between two trading entities. For example, trading
 | `fees.commission` | No | `null` |
 | `fees.omit_sec` | No | `false` |
 | `fees.omit_taf` | No | `false` |
+| `fixed_income.accrued_interest` | No | `0` | `1100.50` |
```

Comment:
shouldn't the last column explain what the field is

Reply:
it does that on the top section of the doc